src/components/header/HomeHeader.vue

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+import { ref, watch } from 'vue'
+import { useDebounce } from '../../custom-hooks/use-debounce'
+import { useParamsStore } from '../../stores/params-store'
+import { defineComponent } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+export default defineComponent({
+  setup() {
+    const paramsStore = useParamsStore()
+    const searchQuery = ref(paramsStore.searchQuery)
+    const debouncedSearchQuery = useDebounce(searchQuery, 1000)
+    const { t } = useI18n()
+    watch(debouncedSearchQuery, (newValue) => {            
+      paramsStore.setSearchQuery(newValue)
+    })
+
+    return {
+      searchQuery,
+      t
+    }
+  }
+})
+</script>
+
+<template>
+  <header
+    class="header-backgound bg-black-600 flex h-96 md:h-[450px] w-full flex-col items-center justify-center bg-auto py-8 md:px-0 lg:p"
+  >
+    <p class="flex justify-center">
+      <a href="https://www.algolia.com/">
+        <img
+          src="/images/algolia.svg"
+          :width="92"
+          :height="24"
+          :alt="t('Algolia_Header_Icon_Alt')"
+        />
+      </a>
+    </p>
+    <p class="my-9 flex justify-center font-thin text-white md:text-3xl lg:text-5xl">
+      {{ t('Header_Title') }}
```

Comment:
```suggestion
      {{ t('header_title') }}
```
Đối với i18n thì e dùng snake case nha, e sửa những chỗ khác nữa nhé

Reply:
done ạ